imsim/atmPSF.py

```diff
@@ -125,13 +132,17 @@ def __init__(self, airmass, rawSeeing, band, boresight, rng,
         self.exptime = exptime
         self.screen_size = screen_size
         self.screen_scale = screen_scale
+        self.exponent = exponent
+        self.opt = None
 
         if save_file and os.path.isfile(save_file):
             self.logger.warning(f'Reading atmospheric PSF from {save_file}')
             self.load_psf(save_file)
         else:
             self.logger.warning('Building atmospheric PSF')
             self._build_atm(kcrit, doOpt, nproc)
+            if _no2k:
+                self.second_kick = galsim.DeltaFunction()
```

Comment:
This feels like the wrong place for this.  I think better to put it in _build_atm, and pass another argument do2k, like we have doOpt.